### PR TITLE
Adding an athens philosophy document

### PR DESCRIPTION
**Pull request by arschles**
I'm attempting here to document generally how we like to work and set some standards so that newcomers know what to expect when they join our community.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,8 +43,11 @@ This project is early and there's plenty of interesting and challenging work to
 If you find a bug or want to fix a bug, we :heart: PRs and issues! If you see an issue
 in the [queue](https://github.com/gomods/athens/issues) that you'd like to work on, please just post a comment saying that you want to work on it. Something like "I want to work on this" is fine.
 
-If you decide to contribute (we hope you do :smile:), the process is familiar and easy if you've used Github before.
-There are no long documents to read or complex setup.
+If you decide to contribute (we hope you do :smile:), the process is familiar and easy if you've used Github before. There are no long documents to read or complex setup.
```
Comment (michalpristas) [inline README.md]: can you attach brians video for first PR: https://www.youtube.com/watch?v=bgSDcTyysRc

File: PHILOSOPHY.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
+# The Athens Philosophy
+
+This document lays out generally how we want to work with each other. It's hard to make a rule or set a guideline for each and every situation that might come up in our community. That's basically predicting the future!
+
+We do of course set some boundaries like the [code of conduct](README#Code-Of-Conduct), but we want to fall back to this document for guidance when we encounter a new situation or question that we need to address.
+
+# Guiding Principles
+
+This is the TL;DR of the whole document! The Athens project has guiding principles:
```
Comment (robjloranger) [inline PHILOSOPHY.md]: maybe a small change here like s/project has guiding principles/project has {some, a few} guiding principles/

File: README.md [hunk 3]
```diff
@@ -43,8 +43,11 @@ This project is early and there's plenty of interesting and challenging work to
...
+
+Before you do start contributing or otherwise getting involved, we want to let you know that we follow a general [philosophy](./PHILOSOPHY.md) in how we work together, and we'd really appreciate you to get familiar with it before you start.
```
Comment (robjloranger) [inline README.md]: maybe s/appreciate you to get familiar/appreciate you getting familiar/ or similar

**Comment by arschles** [comment]
@robjloranger just pushed the grammar fixes. @michalpristas I'm watching the Go OSS contribution video now 😄

**Comment by arschles** [comment]
@michalpristas added the OSS contribution video. I ❤️ it

**Pull request merged**